internal/handlers/common/params_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMultiplyLongSafely(t *testing.T) {
+	t.Parallel()
+
+	for name, tc := range map[string]struct {
+		err              error
+		v1, v2, expected int64
+	}{
+		"Zero": {
+			v1:       0,
+			v2:       1000,
+			expected: 0,
+		},
+		"One": {
+			v1:       42,
+			v2:       1,
+			expected: 42,
+		},
```

Comment:
So if we remove https://github.com/FerretDB/FerretDB/pull/2232/files#diff-a21b8c640c12ecfe3c572f59caf5486544b411403f77500597b373634b17b726R385
at least some test fail.
```
		"Zero": {
			v1:       0,
			v2:       MinInt64,
			expected: MinInt64,
		},
		"One": {
			v1:       1,
			v2:       MinInt64,
			expected: MinInt64,
		},
```

Reply:
Yes, those will fail as `MinInt64` is handled explicitly